src/services/player/autoPlayer.js

```diff
@@ -0,0 +1,36 @@
+// export default
+class AutoPlayer {
+  constructor() {
+    this.id = 'AutoPlayer';
+  }
+
+  generateAutoPlayerAnswer(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1) + min);
+  }
+
+  getAnswer(image, answers) {
+    // console.log(image);
+    // console.log(answers);
+    if (typeof image !== 'string') {
+      throw new Error('image type error');
```

Comment:
I havent noticed those errors in my previous code check, could you please refactor them to be more readable in case that this error occur?

e.g.: `An error occurred while generating answer: type of...`

Reply:
please have in mind that those errors can be read by user or other developer, maybe you will remember what exactly it means, but someone else will have to read and understand code because error message is not clear

Reply:
Done.